### PR TITLE
Add change_url

### PR DESCRIPTION
**Pull request by oufmilo**
## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Package_check results
---
* An automatic package_check will be launch at https://ci-apps-dev.yunohost_url.org/, when you add a specific comment to your Pull Request: "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!"*

### CODE REVIEW

**Comment by oufmilo** [comment]
!testme

**Comment by oufmilo** [comment]
Merge to testing ?

File: scripts/change_url [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#=================================================
+# SPECIFIC MODIFICATIONS
+#=================================================
+ynh_script_progression --message="Applying Misskey specific modifications..." --weight=2
+
+if [ $change_domain -eq 1 ]
+then
+	# Change domain in Misskey configuration
+	ynh_replace_string --match_string="'${old_domain}'" --replace_string="'${new_domain}'" --target_file="${final_path}/config.php"
+fi
```
Comment (ericgaspar) [inline scripts/change_url:99]: Before merging, there is some issues to fix:
- Fail2Ban is not configured in install script ...
-you should recreate an appropriate configuration file: https://github.com/YunoHost-Apps/misskey_ynh/blob/d88816075620c100303d988f5d8bd5af4378f065/scripts/install#L123
- you don't start systemd.service.

**Review by ericgaspar — CHANGES_REQUESTED** [review]

File: scripts/change_url (hunk 1 above)
Comment (oufmilo) [reply scripts/change_url:99]: i'm not sure it's a good idea to change URL with fediverse problem and federation .. i close this PR :+1:

**Pull request closed without merging**